machine_learning_hep/fitting/helpers.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, database: dict, ana_type: str, file_data_name: str, file_mc_n
         self.n_bins1 = len(self.bins1_edges_low)
         self.bin2_name = ana_config.get("var_binning2", None)
         if self.bin2_name is not None:
-            self.bin2_gen_name = ana_config.get("var_binning2_gen")
+            self.bin2_gen_name = ana_config.get("var_binning2")
```

Comment:
Hej @talazare 
Can you comment on why you changed that? To me that doesn't make sense since `self.bin2_gen_name` should definitely refer to the `var_binning2_gen` as defined in the DB.
This change might actually break other analyses and we should **not** change it. What you can do is

```python
self.bin2_gen_name = ana_config.get("var_binning2_gen", self.bin2_name)
```
which means its a fallback to `var_binning2` in case `var_binning2_gen` could not be found

Reply:
Hi, @benedikt-voelkel . This change should be removed. I tried to use this fitter in one particular case, and forgot to change it back, very sorry.